src/components/results/common/utils.ts

```diff
@@ -9,6 +9,10 @@ import { NA_Value } from '../../spreadsheet/utils/cell-renderers';
 
 export const PERMANENT_LIMIT_NAME = 'permanent';
 
+export const MIN_COLUMN_WIDTH = 160;
```

Comment:
move to it to dynamicsimulation ?

Reply:
DONE in https://github.com/gridsuite/gridstudy-app/pull/2499/commits/1ea3c0b9c73d9f412fb210662b2877eb0ddddde1